### ISSUE TITLE
Last open question interpreted as definition

### ISSUE THREAD

**Issue by stemvork** [Yvee1/hascard#6]
When the last card contains an open question, it is interpreted as a definition card. That is, "Avond" is displayed, but the sentence to fill in is not.

```
# Ochtend
De ochtend is voor _werk_.
---
# Middag
De middag is voor _werk_.
---
# Avond
De avond is _NIET|niet_ voor werk.
```

**Comment by stemvork** [comment]
![Schermafbeelding 2020-07-23 om 07 51 24](https://user-images.githubusercontent.com/2034251/88255677-5842a480-ccb9-11ea-8362-c52c19d55367.png)

**Comment by Yvee1** [comment]
I see, thanks for opening the issue, that is quite annoying. I've looked at it briefly and noticed that the problem does not occur if you end the file with a `---`:
```
# Ochtend
De ochtend is voor _werk_.
---
# Middag
De middag is voor _werk_.
---
# Avond
De avond is _NIET|niet_ voor werk.
---
```
So that is a temporary fix. While investigating I also noticed that empty lines at the end cause it to fail; some editors like vim automatically add a new line at the end of the file, so that is a problem. I'll have a look today, and see if I can fix all these parsing issues.

**Comment by stemvork** [comment]
Thanks for the swift reply. Indeed, I tried that and it led to said error (I use vim). I have just learned about "set binary" combined with "set eol". Thanks for looking into the parsing. This is a great workflow :)

**Comment by Yvee1** [comment]
The issue should be fixed now in version 0.1.4.0!
